.github/workflows/docker-image.yml

```diff
@@ -197,7 +197,7 @@ jobs:
         run: |
           echo DEFAULT: E2E test will run with: $E2EBRANCHNAME
           echo "Running on: ${{ steps.branch-name.outputs.current_branch }}"
-          if [ $(git ls-remote --heads https://github.com/mangata-finance/mangata-e2e.git "${{ steps.branch-name.outputs.current_branch }}" --force --quiet  | wc -l) -gt 0 ]; then echo "E2EBRANCHNAME=${{ steps.branch-name.outputs.current_branch }}" >> $GITHUB_ENV; echo "MATCH - OK" ;  fi
+          if [ -n "$(git ls-remote --heads https://github.com/mangata-finance/mangata-e2e.git ${{ steps.branch-name.outputs.current_branch }} --force --quiet)" ]; then echo "E2EBRANCHNAME=${{ steps.branch-name.outputs.current_branch }}" >> $GITHUB_ENV; echo "MATCH - OK" ; elif [ -n "$(git ls-remote --heads https://github.com/mangata-finance/mangata-e2e.git ${{ github.base_ref }} --force --quiet)" ]; then echo "E2EBRANCHNAME=${{ github.base_ref }}" >> $GITHUB_ENV; echo "MATCH - OK" ;  fi
```

Comment:
@goncer my take on branch matchin, idea is to auto select e2e by looking not only for name of the branch from which pr was started but also on the target branch. This way it is quite easy to setup e2e branch for maintanance branch.

What do you think ?

Reply:
super!